web/src/lib/components/settings/Lightning.svelte

```diff
@@ -0,0 +1,100 @@
+<script lang="ts">
+    import { onDestroy, onMount } from 'svelte';
+    import { ErrorHandler, putVerifyLnurl, getProfile } from "$lib/services/api";
+    import { user, token, Info } from "$lib/stores";
+    import Wallets from "$lib/components/notifications/Wallets.svelte";
+    import Loading from "$lib/components/Loading.svelte";
+    import QR from "$lib/components/QR.svelte";
+
+    export let onSave: () => void = () => {};
+
+    let hasLnauthKey: boolean = false;
+
+    let lnurl;
+    let qr;
+    let k1: string | null = null;
+
+    let checkTimeout: ReturnType<typeof setTimeout> | null = null;
+
+    function stopChecking() {
+        if (checkTimeout !== null) {
+            clearTimeout(checkTimeout);
```

Comment:
Passing an invalid ID to clearTimeout() silently does nothing; no exception is thrown. Just in case you want to remove the check.